cmd/process-agent/main_common.go

```diff
@@ -4,11 +4,14 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
+	"github.com/DataDog/datadog-agent/pkg/config/settings"
```

Comment:
Could you please move it into the imports section below.

Reply:
Yeah sorry, I miss these all the time. I also went ahead and fixed it in main.go as well.